helm/thingsboard/values.yaml

```diff
@@ -0,0 +1,284 @@
+    #server: docker.io
+    #pullPolicy: Always
+    #tag: "X.Y.Z"
+  replicaCount: 1
+  ssl:
+    enabled: false
+    secret: kubernetes-secret
+    cert: tls.crt
+    key: tls.key
+  port:
+    number: 8081
+    protocol: TCP
+    name: mqtt
+  timeout: 10000
+  podAnnotations: {}
+  podSecurityContext: {}
+  securityContext: {}
+  service:
+    type: ClusterIP
+    port: 8883
+  resources: {}
+  autoscaling:
+    enabled: false
+    minReplicas: 1
+    maxReplicas: 100
+    targetCPUUtilizationPercentage: 80
+    # targetMemoryUtilizationPercentage: 80
+  nodeSelector: {}
+  tolerations: []
+  affinity: {}
+
+http:
+  kind: StatefulSet
+  image:
+    repository: thingsboard/tb-http-transport
+    # Overrides the global image values
+    #server: docker.io
+    #pullPolicy: Always
+    #tag: "X.Y.Z"
+  replicaCount: 1
+  port:
+    number: 8082
+    protocol: TCP
+    name: http
+  timeout: 10000
+  podAnnotations: {}
+  podSecurityContext: {}
+  securityContext: {}
+  service:
+    type: ClusterIP
+    port: 8082
+  resources: {}
+  autoscaling:
+    enabled: false
+    minReplicas: 1
+    maxReplicas: 100
+    targetCPUUtilizationPercentage: 80
+    # targetMemoryUtilizationPercentage: 80
+  nodeSelector: {}
+  tolerations: []
+  affinity: {}
+
+coap:
+  kind: StatefulSet
+  image:
+    repository: thingsboard/tb-coap-transport
+    # Overrides the global image values
+    #server: docker.io
+    #pullPolicy: Always
+    #tag: "X.Y.Z"
+  replicaCount: 1
+  port:
+    number: 8083
+    protocol: UDP
+    name: coap
+  timeout: 10000
+  podAnnotations: {}
+  podSecurityContext: {}
+  securityContext: {}
+  service:
+    type: ClusterIP
+    port: 8083
+  resources: {}
+  autoscaling:
+    enabled: false
+    minReplicas: 1
+    maxReplicas: 100
+    targetCPUUtilizationPercentage: 80
+    # targetMemoryUtilizationPercentage: 80
+  nodeSelector: {}
+  tolerations: []
+  affinity: {}
+
+webui:
+  kind: Deployment
+  image:
+    repository: thingsboard/tb-web-ui
+    # Overrides the global image values
+    #server: docker.io
+    #pullPolicy: Always
+    #tag: "X.Y.Z"
+  replicaCount: 1
+  port:
+    number: 8084
+    protocol: TCP
+    name: http
+  podAnnotations: {}
+  podSecurityContext: {}
+  securityContext: {}
+  service:
+    type: ClusterIP
+    port: 8084
+  resources: {}
+  autoscaling:
+    enabled: false
+    minReplicas: 1
+    maxReplicas: 100
+    targetCPUUtilizationPercentage: 80
+    # targetMemoryUtilizationPercentage: 80
+  nodeSelector: {}
+  tolerations: []
+  affinity: {}
+
+cassandraInitDB:
+  keyspace: thingsboard
+  replication:
+    class: SimpleStrategy
+    factor: 1
+
+cassandra:
+  # Database can be either postgres or hybrid (entities in postgres and timeseries in cassandra)
+  # for an postgresql database set cassandra.enabled as false
+  # for an hybrid database set cassandra.enabled as true
+  enabled: false
+  replicaCount: 1
+  initDBConfigMap: '{{ .Release.Name }}-cassandra-init-db'
```

Comment:
I think this is invalid name in values?

Reply:
Previous Helm chart also uses this syntax and works, so not that bad. 👍🏻 
Although if we can have the name set in the ConfigMap using  `set` would be much clean.

Reply:
fixed as suggested, setting this value in the configmap in a pre-install hook so it's available for cassandra dependency